recipe/install_base.bat

```diff
@@ -1,9 +1,30 @@
-:: site.cfg should not be defined here.  It is provided by blas devel packages (either mkl-devel or openblas-devel)
+@echo on
 
-COPY %PREFIX%\site.cfg site.cfg
+set "PKG_CONFIG_PATH=%LIBRARY_LIB%\pkgconfig;%LIBRARY_PREFIX%\share\pkgconfig;%BUILD_PREFIX%\Library\lib\pkgconfig"
+if "%blas_impl%" == "openblas" (
+    set "BLAS=openblas"
+)
+else (
+    set "BLAS=mkl-sdl"
+)
 
-%PYTHON% -m pip install --no-deps --ignore-installed --no-build-isolation -v .
-if errorlevel 1 exit 1
+mkdir builddir
+REM -wnx flags mean: --wheel --no-isolation --skip-dependency-check
+"%PYTHON%" -m build -w -n -x ^
```

Comment:
I would use the long names instead of adding a comment to describe the short names.

Reply:
That's fair. That bit is from cf, I will update.

Reply:
updated